solution/ui/regulations/eregs-component-lib/src/components/CollapseButton.vue

```diff
@@ -73,7 +73,7 @@ export default {
         eventbus.on("collapse-toggle", this.toggle);
     },
 
-    beforeDestroy() {
+    beforeUnmount() {
         eventbus.off("collapse-toggle", this.toggle);
```

Comment:
lint autofix: `beforeDestroy` and `destroyed` have been renamed to `beforeUnmount` and `unmounted` in Vue 3